src/main/java/com/sourcegraph/vcs/ConvertUtil.kt

```diff
@@ -1,50 +0,0 @@
-package com.sourcegraph.vcs
-
-import java.net.URI
-
-data class CodebaseName(val value: String)
-
-fun convertGitCloneURLToCodebaseNameOrError(theCloneURL: String): CodebaseName {
-  val cloneURL = theCloneURL.lowercase()
-
-  // Handle common Git SSH URL format
-  val sshUrlRegexMatchResult = Regex("""^[\w-]+@([^:]+):([\w-]+)/([\w-.]+)$""").find(cloneURL)
-  if (sshUrlRegexMatchResult != null) {
-    val (host, owner, repo) = sshUrlRegexMatchResult.destructured
-    return CodebaseName("$host/$owner/${repo.replace(".git$".toRegex(), "")}")
-  }
-
-  var uri = URI(cloneURL)
-  if (uri.scheme == null) {
-    uri = URI("http://$cloneURL")
-  }
-
-  // Handle Azure DevOps URLs
-  if (uri.host?.contains("dev.azure") == true && !uri.path.isNullOrEmpty()) {
```

Comment:
Are all of these cases handled in unit tests on the Agent side?

Reply:
Yes [in this file](https://github.com/sourcegraph/cody/blob/fb5f4c66c21ec0057ae02a6602a0bca9b483289b/lib/shared/src/utils.test.ts). In fact it looks like the tests here were carbon copied from there.